docker/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# enable conda for this shell
+# shellcheck disable=SC1091
+. /opt/conda/etc/profile.d/conda.sh
+
+# activate the environment
+conda activate gadgetron
+
+# exec the cmd/command in this process, making it pid 1
```

Comment:
Nit: I think tini will be PID1

Reply:
Good call. It was a leftover from before we added `tini`. Will remove.